tensorboard/webapp/runs/views/runs_table/runs_table_container.ts

```diff
@@ -173,6 +173,12 @@ function matchFilter(
       filter.filterValues;
     return values.includes(value);
   } else if (filter.type === DomainType.INTERVAL) {
+    // Auto-added to unblock TS5.0 migration
+    //  @ts-ignore(go/ts50upgrade): Operator '<=' cannot be applied to types
+    //  'number' and 'string | number | boolean'.
+    // Auto-added to unblock TS5.0 migration
+    //  @ts-ignore(go/ts50upgrade): Operator '<=' cannot be applied to types
+    //  'string | number | boolean' and 'number'.
```

Comment:
There are two files in the original change. Only one file in this change.

Reply:
The other file is an internal file, not part of our public/GH repo.

Reply:
Whoops, I see now. Makes sense.